modules/core/src/main/kotlin/datamaintain/core/config/DatamaintainConfig.kt

```diff
@@ -207,6 +209,7 @@ enum class CoreConfigKey(override val key: String,
     TAG("tag"),
 
     // FILTER
+    SCAN_FILENAME_REGEX("filter.filename.regex", "(.*)"),
```

Comment:
I find the group here confusing since there is no group extraction. 
```suggestion
    SCAN_FILENAME_REGEX("filter.filename.regex", ".*"),
```